src/pages/index.astro

```diff
@@ -4,6 +4,7 @@ import Home from "@organisms/Home.astro";
 import Speakers from "@organisms/Speakers.astro";
 import Schedule from "@organisms/Schedule.astro";
 import Faq from "@organisms/Faq.astro";
+import Carousel from "@/components/organisms/carousel_js/EmblaCarousel";
```

Comment:
you should make a Carousel.astro and have the carousel, navigation buttons, section container and section title inside of it